api/working-with-extensions/publishing-extension.md

```diff
@@ -87,81 +87,110 @@ First off, follow the documentation to [create your own organization](https://le
 
 1. Click **Create**.
 
-    You'll be presented with your newly created Personal Access Token. **Copy** it to the safe location, you'll need it to [create a publisher](#create-a-publisher).
+    You'll be presented with your newly created Personal Access Token. **Copy** it to the safe location, you'll need it to [create a publisher](#creating-a-publisher).
 
-### Create a publisher
+### Creating a publisher
 
-A **publisher** is an identity who can publish extensions to the Visual Studio Code Marketplace. Every extension needs to include a `publisher` name in its [`package.json` file](/api/references/extension-manifest).
+A **publisher** is an identity that can publish extensions to the Visual Studio Code Marketplace. Every extension should have a `publisher` name in its [`package.json` file](/api/references/extension-manifest).
 
-You can create a new publisher through the Visual Studio Marketplace publisher [management page](https://marketplace.visualstudio.com/manage). The publisher name and publisher display name must be unique. You need to login in with the same Microsoft account you used to create the [Personal Access Token](/api/working-with-extensions/publishing-extension#get-a-personal-access-token) in the previous section.
+To create a pulisher:
 
-Test your publisher's personal access token using [`vsce`](#vsce), while at the same time storing it for later usage:
+1. Go to the [Visual Studio Marketplace publisher management page](https://marketplace.visualstudio.com/manage).
+1. Log in with the same Microsoft account you used to create the [Personal Access Token](/api/working-with-extensions/publishing-extension#getting-a-personal-access-token) in the previous section.
```

Comment:
Since the link is in the same article, you don't need to include the relative file path just the header link